hazelcast-jet-core/src/main/java/com/hazelcast/jet/aggregate/AggregateOperations.java

```diff
@@ -379,15 +381,30 @@ private AggregateOperations() {
      * If the {@code mapFn} returns {@code null}, the item won't be aggregated
      * at all. This allows applying a filter at the same time.
      *
+     * <h4>Example</h4>
+     *
+     * <pre>{@code
+     *     // calculate set of surnames in each city
+     *     p.drawFrom(...)
```

Comment:
I think having the pipeline here creates more noise, we could just have the AggregateOperations part, what do you think?